hubspot3/test/test_deals.py

```diff
@@ -1,14 +1,25 @@
 """
-testing hubspot3.deals
+Testing hubspot3.deals
+This file tests the methods in the hubspot3.deals file
 """
 import pytest
 from hubspot3.deals import DealsClient
 from hubspot3.error import HubspotNotFound, HubspotBadRequest
 from hubspot3.test.globals import TEST_KEY
-
+from hubspot3 import deals
+import json
+from unittest.mock import Mock
 
 DEALS = DealsClient(api_key=TEST_KEY)
```

Comment:
```suggestion
deals_client = deals.DealsClient(api_key=TEST_KEY)
```
Também recomendo usar uma variável em lower-case aqui, o padrão para letras maiúsculas é normalmente em constantes e não objetos.